extensions/src/main/java/org/pentaho/platform/plugin/services/importer/RepositoryFileImportFileHandler.java

```diff
@@ -49,10 +50,13 @@
  */
 public class RepositoryFileImportFileHandler implements IPlatformImportHandler {
 
+  private static final List<String> KNOWN_EXTENSIONS = Arrays.asList( "prpt", "prpti", "xdash",
+          "xaction", "xanalyzer", "kjb", "ktr", "xcdf", "xmi", "cda", "xreportspec" );
```

Comment:
This should get read from the system configurations, not hardcoded here. Right? I mean, the accepted extensions are defined in importExport.xml or whatever that is. So we need to query the system to know which file extensions are known.